### PR TITLE
Add nav bar and implement Google styling CSS / HTML guidelines

### PR DESCRIPTION
**Pull request by saraholiviawu**
Will change colors and background of nav bar in the future. Current features: hovering over any of the options on the right side of the navigation bar will underline the text.

### CODE REVIEW

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -1,9 +1,49 @@
...
+}
+
+.navbar-link {
+    height:22px;
+    border-bottom: 1px solid transparent;
+    transition: border-bottom 0.5s; 
+}   
+
+.navbar-link a {
+    color:rgb(21, 74, 154);
```
Comment (sabadas) [inline portfolio/src/main/webapp/style.css:32]: Don't know if this is possible, but does css allow variables? I see this color is used in another place, so maybe you can use a variable so you only have to change one spot if changes need to be made?

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -1,9 +1,49 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+
+
+/* Navigation bar */
+.navbar {
+    /*display: flex;
+    justify-content: space-between; */
```
Comment (sabadas) [inline portfolio/src/main/webapp/style.css:6]: Is this comment meant to still be there?

Comment (saraholiviawu) [reply portfolio/src/main/webapp/style.css:4]: No, good catch thanks!

Comment (saraholiviawu) [reply portfolio/src/main/webapp/style.css:4]: Will delete in the following commit

File: portfolio/src/main/webapp/style.css (hunk 1 above)
Comment (saraholiviawu) [reply portfolio/src/main/webapp/style.css:30]: I believe so, I will look into it and add a variable if possible in the following commit

**Pull request merged**